packages/react-core/src/components/MultipleFileUpload/MultipleFileUploadStatusItem.tsx

```diff
@@ -54,6 +55,8 @@ export interface MultipleFileUploadStatusItemProps extends React.HTMLProps<HTMLL
   progressAriaLiveMessage?: string | ((loadPercentage: number) => string);
   /** Unique identifier for progress. Generated if not specified. */
   progressId?: string;
+  /** @beta Additional content related to the status item, intended to be dynamically rendered content such as status messages. */
+  helperText?: React.ReactNode;
```

Comment:
maybe rename this to `progressHelperText` to match the other progress relate props